src/test/java/uk/gov/hmcts/befta/dse/ccd/CcdBeftaUtilsTest.java

```diff
@@ -21,9 +21,9 @@
  */
 class CcdBeftaUtilsTest {
 
-    public static final String DEFAULT_DEFINITIONS_PATH_JSON = "src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/valid/CCD_CNP_27";
-    public static final String DEFAULT_DEFINITIONS_PATH_EXCEL = "src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/excel/CCD_CNP_27.xlsx";
-    public static final String DEFAULT_DEFINITIONS_PATH_FOLDER = "src/main/resources/uk/gov/hmcts/befta/dse/ccd/definitions/excel/AUTOTEST1";
+    public static final String DEFAULT_DEFINITIONS_PATH_JSON = "src/test/resources/uk/gov/hmcts/befta/dse/ccd/definitions/valid/CCD_CNP_27";
```

Comment:
A single constant to avoid repetition of the paths ?

`src/test/resources/uk/gov/hmcts/befta/dse/ccd/definitions/valid`
`src/test/resources/uk/gov/hmcts/befta/dse/ccd/definitions/excel`

Reply:
@sullivangareth 
May I just keep to the current standards in this sense? Could you please approve  the PR otherwise, if you find it ok to go?